packages/color-block/src/ColorBlockDropsView.tsx

```diff
@@ -0,0 +1,127 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { FC } from 'react';
+
+import {
+    Button,
+    ButtonStyle,
+    IconPlus,
+    IconSize,
+    IconTrash,
+    RichTextEditor,
+    Tooltip,
+    TooltipPosition,
+} from '@frontify/fondue';
+import { joinClassNames, useGuidelineDesignTokens } from '@frontify/guideline-blocks-shared';
+
+import { ColorBlockDropsViewProps } from './types';
+import { copyToClipboard } from './utilities/copyToClipboard';
+import { mapColorSpaces } from './helpers/mapColorSpaces';
+import { ColorsBlockColorPicker } from './components/ColorsBlockColorPicker';
+
+export const ColorBlockDropsView: FC<ColorBlockDropsViewProps> = ({
+    colors,
+    colorSpaces,
+    isEditing,
+}: ColorBlockDropsViewProps) => {
+    const { designTokens } = useGuidelineDesignTokens();
+
+    return (
+        <div className="tw-flex tw-flex-wrap tw-gap-y-8">
+            {colors?.map((item) => (
+                <div key={item} className="tw-group tw-flex tw-flex-col tw-items-center tw-w-1/6">
+                    <div
+                        className={joinClassNames([
+                            'tw-relative tw-w-[100px] tw-h-[100px] tw-rounded-full tw-mb-3 tw-bg-black-warm tw-transition-all',
+                            isEditing && 'group-hover:tw-shadow-inset-hover-strong',
+                            !isEditing && 'group-hover:tw-shadow-inset-hover-weak',
+                        ])}
+                    >
+                        {isEditing && (
+                            <div
+                                className={joinClassNames([
+                                    'tw-absolute tw-hidden tw-top-[-0.25rem] tw-right-[-0.25rem]',
+                                    isEditing && 'group-hover:tw-block',
+                                ])}
+                            >
+                                <Button
+                                    icon={<IconTrash size={IconSize.Size20}></IconTrash>}
```

Comment:
you can use `<IconTrash size={IconSize.Size20} />` in components. Same for `<Button .... />`